src/Microsoft.IdentityModel.Protocols.OpenIdConnect/OpenIdConnectProtocolValidator.cs

```diff
@@ -540,12 +540,12 @@ protected virtual void ValidateCHash(OpenIdConnectProtocolValidationContext vali
         /// <summary>
         /// Validates the 'token' according to http://openid.net/specs/openid-connect-core-1_0.html
         /// </summary>
-        /// <param name="validationContext">a <see cref="OpenIdConnectProtocolValidationContext"/> that contains the protocol message to validate.</param>
-        /// <exception cref="ArgumentNullException">if 'idToken' is null.</exception>
-        /// <exception cref="ArgumentNullException">if 'validationContext' is null.</exception>
-        /// <exception cref="OpenIdConnectProtocolInvalidAtHashException">if the validationContext contains a 'token' and there is no 'at_hash' claim in the id_token.</exception>
-        /// <exception cref="OpenIdConnectProtocolInvalidAtHashException">if the validationContext contains a 'token' and the 'at_hash' claim is not a string in the 'id_token'.</exception> 
-        /// <exception cref="OpenIdConnectProtocolInvalidCHashException">if the 'at_hash' claim in the 'idToken' does not correspond to the 'token' in the <see cref="OpenIdConnectMessage"/> response.</exception> 
+        /// <param name="validationContext">A <see cref="OpenIdConnectProtocolValidationContext"/> that contains the protocol message to validate.</param>
+        /// <exception cref="ArgumentNullException">If 'idToken' is null.</exception>
+        /// <exception cref="ArgumentNullException">If 'validationContext' is null.</exception>
+        /// <exception cref="OpenIdConnectProtocolInvalidAtHashException">If the validationContext contains a 'token' and there is no 'at_hash' claim in the id_token.</exception>
+        /// <exception cref="OpenIdConnectProtocolInvalidAtHashException">If the validationContext contains a 'token' and the 'at_hash' claim is not a string in the 'id_token'.</exception> 
+        /// <exception cref="OpenIdConnectProtocolInvalidCHashException">If the 'at_hash' claim in the 'idToken' does not correspond to the 'token' in the <see cref="OpenIdConnectMessage"/> response.</exception> 
```

Comment:
change: 'idToken' to 'id_token'
change: 'token' to 'access_token'
change: OpenIdConnectProtocolInvalidCHashException to OpenIdConnectProtocolInvalidAtHashException